kratos/python/add_properties_to_python.cpp

```diff
@@ -23,13 +22,16 @@
 #include "includes/condition.h"
 #include "includes/properties.h"
 #include "includes/constitutive_law.h"
-#include "python/add_mesh_to_python.h"
+#include "python/add_properties_to_python.h"
 #include "python/containers_interface.h"
 
-namespace Kratos
-{
-namespace Python
+using AcccessorBindType = std::unique_ptr<Kratos::Accessor>;
+
+PYBIND11_MAKE_OPAQUE(AcccessorBindType);
```

Comment:
Shouldn't we define this once within the `Kratos::Python` namespace? As it is right now it is both here and in the `add_accessors_to_python.cpp`.

Reply:
Yep but I don't know a clean way to do it. It needs to be defined in the python layer. If we put in either add_properties.h or add_accessor.h then we have to import the other in the file does not belong to. If we really want to have it defined one I would put it in the properties.

In any case as before, needs to be outside Kratos namespace

Reply:
btw, you have too many cs in the name